critiquebrainz/frontend/external/bookbrainz_db/test/literary_work_test.py

```diff
@@ -33,4 +33,4 @@ def test_fetch_edition_groups_for_works(self):
         self.assertEqual(edition_group_bbids_1[0], "02ae4cfc-6412-4693-93b1-e24dce5e31f9")
 
         edition_group_bbids_2 = literary_work.fetch_edition_groups_for_works(self.bbid3)
-        self.assertEqual(len(edition_group_bbids_2), 0)
+        self.assertEqual(len(edition_group_bbids_2), 1)
```

Comment:
:thinking: It seems that this test was passing at the time it has originally been added at https://github.com/metabrainz/critiquebrainz/pull/459/commits/5faec6ba8df30c9b091f475dab211677fb6ba718#diff-f5a1981beb9671ab22e01b3d6fcd412420ce491dfc52668cbb777ef29d1c75b0

1. It is not clear why it broke since then.
2. It seems that changing this value to `1` makes it a quasi-duplicate of the first assertion on `len(edition_group_bbids_1)`, so the test might no longer be covering the case it was supposed to cover in the first place.

Reply:
For testing functionalities for depending on BookBrainz's Database, since we don't have any test dump for BB. So we use the latest BB Dump. Since this data has changed in the dump, therefore this test failed.

Reply:
Thank you, it answers to the 1st point. For the 2nd point, I suggest to replace the current work with [a work that has currently no edition](https://bookbrainz.org/work/14b910e7-c003-491b-923c-eeadd272d29b) so as to keep testing that this function returns `0` in such case.

`self.bbid4 = "14b910e7-c003-491b-923c-eeadd272d29b"`

Reply:
Thanks for addressing this in https://github.com/metabrainz/critiquebrainz/pull/487/commits/bee002dec3bacb9efe86c66316ed81ebea252326